### ISSUE TITLE
Training details on CIFAR-10

### ISSUE THREAD

**Issue by pradipto111** [facebookresearch/deit#165]
I have a few doubts regarding training and teacher-student combination from scratch.
The paper 'Training data-efficient image transformers & distillation through attention' by Touvron et. al. (https://arxiv.org/pdf/2101.00590v1.pdf) on p13 under the section 'Comparison vs training from scratch' claims to have achieved 98.0% accuracy by training RegnetY-16GF from scratch and 98.5% for Deit-B with Distillation.

I would like to know, which teacher was used for the Deit-B distillation run in the above experiments. If it is RegnetY-16GF, was the teacher also trained from scratch on CIFAR-10?

Additionally, it would be of great help if you could provide the implementation details for the runs claimed above (Both RegnetY-16GF and Deit-B distillation) which include learning rate, epochs, batch size, and other relevant details.
